code/modules/reagents/chemistry/recipes/others.dm

```diff
@@ -101,10 +101,10 @@
 	min_temp = 374
 	result_amount = 1
 
-/datum/chemical_reaction/plastication/on_reaction(datum/reagents/holder)
```

Comment:
This line is the only thing that needs to be changed, to:

`/datum/chemical_reaction/plastic_polymers/on_reaction(datum/reagents/holder, created_volume)`